packages/scaffold-config/src/component-index-template.ts

```diff
@@ -0,0 +1,28 @@
+import dedent from 'dedent'
+
+const componentIndexHtmlGenerator = (headModifier: string = '', bodyModifier: string = '') => {
+  return () => {
+    const template = dedent`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Components App</title>
+        ${headModifier}
+      </head>
+      <body>
+        ${bodyModifier}
+        <div data-cy-root></div>
+      </body>
+    </html>
+  `
+
+    // If the framework returns an empty string for either of the modifiers,
+    // strip out the empty lines
+    return template.replace(/\n {4}\n/g, '\n')
```

Comment:
I wonder if we should be using https://nodejs.org/api/os.html#oseol

```sh
os.EOL // =>
    \n on POSIX
    \r\n on Windows
```

I think this would fail on windows?

Reply:
Hm. I don't think this is the right place for that - if we scaffold files using OS EOL separators, then that would be done in the `scaffoldFile` method in `data-context/src/actions/WizardActions.ts`. Then it would be in one place, applying to all files we scaffold to disk.

I don't think we should do that, though. Several other methods in that same file (`scaffoldFixtures` and `scaffoldConfig` for example) use `\n` as well. It'd be a bit of a change.